test/integration/grpcServer/handlers/core/getLastStateTransitionHashHandlerFactory.js

```diff
@@ -42,20 +42,68 @@ describe('getLastStateTransitionHashHandlerFactory', () => {
     );
   });
 
+  it('should throw an error if userId is not a buffer', function it(done) {
+    const userId = 'someStringId';
+
+    const call = new GrpcCallMock(this.sinon, {
+      userId,
+    });
+
+    const callback = (e, v) => {
+      try {
+        expect(v).to.equal(null);
+        expect(e).to.be.an.instanceOf(InvalidArgumentError);
+        expect(e.getMessage()).to.equal('Invalid argument: userId is not a buffer');
+
+        done();
+      } catch (error) {
+        done(error);
+      }
+    };
+
+    coreAPIMock.getUser.resolves(undefined);
+
+    getLastStateTransitionHashHandler(call, callback);
+  });
+
+  it('should throw an error if userId is not of correct length', function it(done) {
+    const userId = Buffer.from('someStringId');
+
+    const call = new GrpcCallMock(this.sinon, {
+      userId,
+    });
+
+    const callback = (e, v) => {
```

Comment:
What if callback won't be called? You can make it spy and check that it must be called.